.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,5 +25,6 @@ assignees: ''
 #### Environment:
 - Kubernetes version (use `kubectl version --short`):
 - Istio version (use `istioctl version`):
+- APIClarity version (use `kubectl -n apiclarity exec deploy/apiclarity -- ./backend version`)
```

Comment:
Do we want to make a CLI for apiclarity? could be nice

Reply:
Also thought about it - could be nice but not sure it worth only for printing version
We can make all the installation process handled by the CLI and convert all the bash logic to code - need to think about it.